io.github.clash_for_windows.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.clash_for_windows
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+base: org.electronjs.Electron2.BaseApp
+base-version: 21.08
+sdk: org.freedesktop.Sdk
+command: cfw.sh
+
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --socket=wayland
+- --socket=system-bus
```

Comment:
does it really support wayland? Then should be `fallback-x11` and not `x11`.
Also `--socket=system-bus` is likely incorrect and unreasonable. You should determine which bus names are needed.

Reply:
I would test fallback-x11, and specify exactly which bus to use.